<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectDialogEdit</class>
 <widget class="QDialog" name="ConnectDialogEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>243</width>
    <height>147</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Server</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="qliName">
     <property name="text">
      <string>&amp;Name</string>
     </property>
     <property name="buddy">
      <cstring>qleName</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="qleName"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="qliServer">
     <property name="text">
      <string>A&amp;ddress</string>
     </property>
     <property name="buddy">
      <cstring>qleServer</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="qleServer"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="qliPort">
     <property name="text">
      <string>&amp;Port</string>
     </property>
     <property name="buddy">
      <cstring>qlePort</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="qlePort"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="qliUsername">
     <property name="text">
      <string>&amp;Username</string>
     </property>
     <property name="buddy">
      <cstring>qleUsername</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="qleUsername"/>
   </item>
   <item row="4" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>qleName</tabstop>
  <tabstop>qleServer</tabstop>
  <tabstop>qlePort</tabstop>
  <tabstop>qleUsername</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConnectDialogEdit</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>112</x>
     <y>123</y>
    </hint>
    <hint type="destinationlabel">
     <x>50</x>
     <y>198</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConnectDialogEdit</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>191</x>
     <y>128</y>
    </hint>
    <hint type="destinationlabel">
     <x>224</x>
     <y>268</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
